05-UI-ReactJS/source/movieHandler.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import {connect} from "react-redux";
+import {handleMovies} from "./redux/reducer";
+import {deleteMovie} from "./redux/action";
+import store from "./store";
+import Movie from "./movie";
+import MovieList from "./movieList";
+
+class MoviesHandler extends React.Component {
+  constructor (props) {
+    super(props);
+  }
+
+  render () {
+    return (
+      <div>
+        <Movie />
+        <MovieList movies={this.props.movies} onDelete={this.deleteMovie}/>
```

Comment:
As you are using redux, you could connect the store to the MovieList and don't pass it as a prop.